app/ide-desktop/lib/dashboard/src/components/dashboard/KeyboardShortcut.tsx

```diff
@@ -59,6 +59,16 @@ const MODIFIER_MAPPINGS: Readonly<
   /* eslint-enable @typescript-eslint/naming-convention */
 }
 
+const KEY_CHARACTER: Readonly<Record<string, string>> = {
+  // The names come from a third-party API (the DOM spec) and cannot be changed.
+  /* eslint-disable @typescript-eslint/naming-convention */
+  ArrowDown: '↓',
+  ArrowUp: '↑',
+  ArrowLeft: '←',
+  ArrowRight: '→',
+  /* eslint-enable @typescript-eslint/naming-convention */
+} satisfies Partial<Record<inputBindingsModule.Key, React.ReactNode>>
```

Comment:
Using ‘ReactNode’ here means that we can use null and undefined as a value. Is it intended?

also: we user here both satisfy and direct type declaration, do we need both?

Reply:
Nice solution, I've been using  smth like `inputBindingsModule.Key | (string&{})` all my life for that )